tests/test_bootstrap.py

```diff
@@ -100,32 +104,106 @@ def test_build_dist_dirs(self):
             bs.get_common_dir().endswith("pythonforandroid/bootstraps/common")
         )
 
-    def test_list_bootstraps(self):
+    def test__cmp_bootstraps_by_priority(self):
+        # Test service_only has higher priority than sdl2:
+        self.assertTrue(_cmp_bootstraps_by_priority(
+            Bootstrap.get_bootstrap("service_only", self.ctx),
+            Bootstrap.get_bootstrap("sdl2", self.ctx)
+        ) < 0)
```

Comment:
Awesome thanks!
Then we can write a simple test for covering the alphabetic sorting `return (a.name - b.name)  # alphabetic sort for determinism`
So if basically if I understood correctly, we just need to comp two bootstraps that're not in `default_recipe_priorities` list. And if we don't have any, we can just patch the list to make it empty.
Would that make sense?

Reply:
Ah, neat idea! I'll add that, thanks :heart_eyes: